docs/pydoc/config/evaluation_api.yml

```diff
@@ -3,7 +3,8 @@ loaders:
     search_path: [../../../haystack/evaluation]
     modules:
       [
-        "eval_run_result"
+        "base",
```

Comment:
nice 👍🏽 

in my opinion, this is the approach to organizing abstractions - we could think about using it on every bit of code that has abstractions

Reply:
Slightly unrelated, but we should refrain from introducing abstractions, like in this case. I would have just gone with `EvaluationRunResult` and added an abstraction later if and when needed, possibly with a protocol, not an abstract class. In that case, we wouldn't have had this problem :)

Reply:
I agree that the abstractions here was too much and we can always add them later if needed.